src/scss/main/_about_methot-section.scss

```diff
@@ -0,0 +1,162 @@
+.about-methot-text-sm {
+  @include media-breakpoint-up(xl) {
+    display: none;
```

Comment:
is it possible to use bootstrap atomic classes instead? (for instance: 'hide-xl')